Validation/RecoTrack/python/MultiTrackValidatorGenPs_cfi.py

```diff
@@ -6,7 +6,8 @@
 from SimTracker.TrackAssociation.CosmicParametersDefinerForTP_cfi import *
 from Validation.RecoTrack.MTVHistoProducerAlgoForTrackerBlock_cfi import *
 
-multiTrackValidatorGenPs = cms.EDAnalyzer(
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+multiTrackValidatorGenPs = DQMEDAnalyzer(
     "MultiTrackValidatorGenPs",
```

Comment:
@schneiml Now that also #21794 got merged, will you (=DQM) take care of rolling this one back to `cms.EDAnalyzer` (or whatever is used for `DQMGlobalEDAnalyzer`) at the latest when you get to the `EDAnalyzer`->`EDProducer` change for `DQMEDAnalyzer`?